### DISCUSSION TITLE
JST EH wiring

### DISCUSSION THREAD

**Discussion by jsapede** [revk/xESP32-Faikout#279] [Q&A]
hello all,
still havent recieved my JST EH connector from aliexpress
got some wires at home for esp32, could someone confirm that the following witring sheme is ok ? or has it to be inverted ? 

![image](https://github.com/revk/ESP32-Faikin/assets/66303110/0702ccae-f373-4b17-b664-c4937b87ec2a)

**Comment by revk** [comment]
Looks right

**Comment by jsapede** [comment]
thanks for the answer , and sorry to have bothered, just found the photo of the original wiring :
![image](https://github.com/revk/ESP32-Faikin/assets/66303110/ec44bed4-c60d-47fc-a148-20a436c5480c)

**Reply by jsapede** [reply]
One more question
On the original it only has 4 wires. Is the 5th one necessary ?

**Reply by revk** [reply]
Not usually needed.

**Reply by jsapede** [reply]
since i havent found it in the docs, would it be possible to have a wire mapping inside the doc and a control pocedure to be sure that nothing is inverted? or maybi it's in the doc and i haven't found it

**Reply by revk** [reply]
It says it is 1:1 from S21 to Faikin.

**Reply by revk** [reply]
<img width="632" alt="Screenshot 2024-03-28 at 15 14 25" src="https://github.com/revk/ESP32-Faikin/assets/996983/24240bb7-0480-4ca2-90bd-3937a149f5a9">

**Reply by jsapede** [reply]
yes but reverted is also 1 to 1 !
how to be sure of the identification the good disposition ?

**Reply by revk** [reply]
What, no, 1:1 is a one to one match of pin to pin, i.e. pin 1 to pin 1, pin 2 to pin 2, etc. I really don't think anyone could think otherwise!

**Reply by jsapede** [reply]
one on one seems ok, depending the way you look at it !
👍 
![PXL_20240329_165021810](https://github.com/revk/ESP32-Faikin/assets/66303110/6d24239c-2c4b-4d1f-89e9-1b55760321ca)
